### DISCUSSION TITLE
Illustration (for CAA abstract but as general illustration)

### DISCUSSION THREAD

**Discussion by simoncarrignon** [acortell3/archaeoriddle#21] [General]
I would be nice to have an illustration that summarize the idea of the project. We could add it to the abstract for CAA and in all our future communications about the project (websites, etc...).  

After Enrico mentioned it I played a bit with [rayshader](https://www.rayshader.com/index.html)  and mixed some of the plot already  available 
![image](https://user-images.githubusercontent.com/4749455/159337017-3293e895-46dd-404e-b215-87c18be25e3f.png)

Very random and ugly, but was just to illustrate what I had in mind (and play with rayshader :D). I'm sure we can do something nicer and meaningful!

**Comment by acortell3** [comment]
Cool! What if we focus on the hexagon produced by rayshader and we add some kernels with two different colours to it, as the population spread? Do you have the file that you used to produce the hexagon, so I can play around and try it?

**Comment by simoncarrignon** [comment]
Yeah it's on the git [here](https://github.com/acortell3/collaborabm/blob/main/doc/bookdown/04_populationgrowth.Rmd), hexagon generated in this chunk :
https://github.com/acortell3/collaborabm/blob/d26aff004cf33e5a99f13bce44053f7d6c9bbc6c/doc/bookdown/03_generatelandscape.Rmd#L144-L167

It compiles on my machine but not on the jomon  `git` user yet, will check that why later.

**Comment by simoncarrignon** [comment]
Had some trouble compiling all on jomon, in part due tylermorganwall/rayshader#222  but it now should be good and http://arch-6kszmf3.arch.private.cam.ac.uk/collaborabm/ should be up to date

**Comment by acortell3** [comment]
Wow... rayshader looks cool indeed... We have to sit and put things together! I'll back tomorrow (I hope!)
